src/tests/stubs/FuseTestEnv.ts

```diff
@@ -151,6 +151,10 @@ export class FuseTestEnv {
 		return this.scripts.get(name);
 	}
 
+	public getScripts() {
```

Comment:
Why don't you just make a getter?

Reply:
java background ;-))

Reply:
It's a public field anyways, so actually... what's the point of this method? :)